scripts/env-vars-setup.cmd

```diff
@@ -0,0 +1,109 @@
+:: IoTHub Info can be found in the Azure portal at:
+:: {Your IoT Hub} > Endpoints > events
+::
+:: Example:
+:: PCS_IOTHUBREACT_HUB_NAME "iothub-test123"
+:: PCS_IOTHUBREACT_HUB_ENDPOINT "Endpoint=sb://iothub-foo-iothub-bar-12345-12345.servicebus.windows.net/;SharedAccessKeyName=iothubowner;SharedAccessKey=12345="
+:: PCS_IOTHUBREACT_HUB_PARTITIONS "4"
+:: -----------------------------------------------------------------------------
+SETX PCS_IOTHUBREACT_HUB_NAME ""
+SETX PCS_IOTHUBREACT_HUB_ENDPOINT ""
+SETX PCS_IOTHUBREACT_HUB_PARTITIONS ""
+
+:: -----------------------------------------------------------------------------
+:: EventHub Info can be found in the Azure portal at:
+:: {Your EventHub} > Shared access policies > RootManageSharedAccessKey
+::
+:: Example:
+:: PCS_EVENTHUB_CONNSTRING "Endpoint=sb://eventhub-foo-eventhub-bar-12345-12345.servicebus.windows.net/;SharedAccessKeyName=RootManageSharedAccessKey;SharedAccessKey=12345="
+:: PCS_EVENTHUB_NAME "eventhub-test123"
+:: -----------------------------------------------------------------------------
+SETX PCS_EVENTHUB_CONNSTRING ""
+SETX PCS_EVENTHUB_NAME ""
+
+:: -----------------------------------------------------------------------------
+:: Storage Account information:
+:: {Your storage account} > Access keys
+::
+:: Example: 
+:: PCS_IOTHUBREACT_AZUREBLOB_ACCOUNT "storagetest123"
+:: PCS_IOTHUBREACT_AZUREBLOB_KEY "12345...6789=="
+:: PCS_ASA_DATA_AZUREBLOB_ACCOUNT "storagetest123"
+:: PCS_ASA_DATA_AZUREBLOB_KEY "12345...6789=="
+:: -----------------------------------------------------------------------------
+SETX PCS_IOTHUBREACT_AZUREBLOB_ACCOUNT ""
+SETX PCS_IOTHUBREACT_AZUREBLOB_KEY ""
+SETX PCS_ASA_DATA_AZUREBLOB_ACCOUNT ""
+SETX PCS_ASA_DATA_AZUREBLOB_KEY ""
+
+:: -----------------------------------------------------------------------------
+:: Storage Endpoint Suffix: 
+:: {Your storage account} > Properties > Primary Blob Service Endpoint > value
+:: in the URI after the "blob." (eg. `core.windows.net`)
+::
+:: Example:
+:: PCS_IOTHUBREACT_AZUREBLOB_ENDPOINT_SUFFIX "core.windows.net"
+:: PCS_ASA_DATA_AZUREBLOB_ENDPOINT_SUFFIX "core.windows.net"
+:: -----------------------------------------------------------------------------
+SETX PCS_IOTHUBREACT_AZUREBLOB_ENDPOINT_SUFFIX ""
+SETX PCS_ASA_DATA_AZUREBLOB_ENDPOINT_SUFFIX ""
+
+:: -----------------------------------------------------------------------------
+:: DocumentDb Conn String:
+:: {Your Document DB} > Keys > Primary Connection String
+:: Note: use this value for all 3 environment variables below
+::
+:: Example:
+:: PCS_STORAGEADAPTER_DOCUMENTDB_CONNSTRING=AccountEndpoint "https://documentdb-test123.documents.azure.com:443/;AccountKey=12345==;"
+:: PCS_TELEMETRY_DOCUMENTDB_CONNSTRING=AccountEndpoint "https://documentdb-test123.documents.azure.com:443/;AccountKey=12345==;"
+:: -----------------------------------------------------------------------------
+SETX PCS_STORAGEADAPTER_DOCUMENTDB_CONNSTRING ""
```

Comment:
We also have:
PCS_STREAMANALYTICS_DOCUMENTDB_CONNSTRING
PCS_TELEMETRYAGENT_DOCUMENTDB_CONNSTRING

Reply:
Are these required?

Reply:
I don't think so. That's why I didn't add them. We can add them on demand.

Reply:
Are we using PCS_STREAMANALYTICS_DOCUMENTDB_CONNSTRING and PCS_TELEMETRYAGENT_DOCUMENTDB_CONNSTRING. Why are these not there in the main repo ?

Reply:
If they aren't in the main repo, then we can leave them out.

Reply:
done